playroom/snippets.tsx

```diff
@@ -623,6 +623,14 @@ const titlesSnippets: Array<Snippet> = [
     },
 ].map((snippet) => ({...snippet, group: 'Titles'}));
 
+const tagSnippets: Array<Snippet> = ['promo', 'active', 'inactive', 'success', 'warning', 'error'].map(
```

Comment:
You did Almendruco's trick and now you can't put the first letter in uppercase in snippets, right? 😬

Reply:
sure you can

Reply:
done